### PR TITLE
[RHACS] Added CLI enhancements for 3.68

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Added CLI enhancements details for 3.68

<hr>

NOTE:

- This PR is for RHACS docs.
- It doesn't require any special labels or milestones.

### CODE REVIEW

File: modules/using-cli.adoc [hunk 1]
```diff
@@ -122,6 +122,81 @@ You can remove them by running the `delete-sensor.sh` script from the Sensor ins
 [id="check-policy-compliance_{context}"]
 == Checking policy compliance
 
+You can use the `roxctl` CLI to check deployment YAML files and images for policy compliance.
+
+[discrete]
+=== Configuring output format
+When you check policy compliance by using the `deployment check`, `image check`, or `image scan` commands, you can specify the output format by using the `-o` option. This option determins how the output of a command is displayed in the terminal.
```
Comment (kalexand-rh) [inline modules/using-cli.adoc:129]:
```suggestion
When you check policy compliance by using the `deployment check`, `image check`, or `image scan` commands, you can specify the output format by using the `-o` option. This option determines how the output of a command is displayed in the terminal.
```

File: modules/using-cli.adoc [hunk 2]
```diff
@@ -122,6 +122,81 @@ You can remove them by running the `delete-sensor.sh` script from the Sensor ins
...
+
+You can change the output format by adding the `-o` option to the command and specifying the format as either `json`, `table`, `csv`, or `junit`.
```
Comment (kalexand-rh) [inline modules/using-cli.adoc:131]:
```suggestion
You can change the output format by adding the `-o` option to the command and specifying the format as `json`, `table`, `csv`, or `junit`.
```

File: modules/using-cli.adoc [hunk 3]
```diff
@@ -122,6 +122,81 @@ You can remove them by running the `delete-sensor.sh` script from the Sensor ins
...
+[source,terminal]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" \
+  deployment check -file =<yaml_filename> \
+  -o csv
+----
+
+[NOTE]
+====
+When you do not specify the `-o` option for the output format:
```
Comment (kalexand-rh) [inline modules/using-cli.adoc:143]: It's better to introduce that a list is coming:

```suggestion
When you do not specify the `-o` option for the output format, the following default behavior is used:
```
If you take this suggestion, you can simplify the list items a bit, like in the next suggestion.

File: modules/using-cli.adoc [hunk 4]
```diff
@@ -122,6 +122,81 @@ You can remove them by running the `delete-sensor.sh` script from the Sensor ins
...
+
+* The default output format for the `deployment check` and the `image check` commands is `table`.
```
Comment (kalexand-rh) [inline modules/using-cli.adoc:145]:
```suggestion
* The format for the `deployment check` and the `image check` commands is `table`.
```

File: modules/using-cli.adoc [hunk 5]
```diff
@@ -122,6 +122,81 @@ You can remove them by running the `delete-sensor.sh` script from the Sensor ins
...
+* The default output format for the `image scan` command is `json`. This is the old JSON format output for compatibility with older versions of the CLI. To get the output in the new JSON format, specify the option with format, as `-o json`.
+====
+
+Depending on the format you select, there are different options available to configure the output. The following tabele lists the options for each format.
```
Comment (kalexand-rh) [inline modules/using-cli.adoc:149]: ISG prefers avoiding "there are," so I suggest something like:
```suggestion
Different options are available to configure the output that depend on the format you select. The following table lists the options for each format.
```

File: modules/using-cli.adoc [hunk 6]
```diff
@@ -122,6 +122,81 @@ You can remove them by running the `delete-sensor.sh` script from the Sensor ins
...
+|Use this option to specify custom headers.
+
+|`--merge-output`
+|Use this options to merge table cells that have the same value.
+
+|`--no-header`
+|Use this option to omit the header row from the output.
+
+|`--row-jsonpath-expressions`
+a|Use this option to specify link:https://github.com/tidwall/gjson[GJSON paths] to select specific items from the output. For example, to get the *Policy name* and *Severity* for a deplyment check, use the following command:
```
Comment (kalexand-rh) [inline modules/using-cli.adoc:172]:
```suggestion
a|Use this option to specify link:https://github.com/tidwall/gjson[GJSON paths] to select specific items from the output. For example, to get the *Policy name* and *Severity* for a deployment check, use the following command:
```
OCP has a blanket ban on github links without clearance by support. (They're concerned about the appearance of extending supportability to uncontrolled sources.) Consider if additional approval would be beneficial here.

File: modules/using-cli.adoc [hunk 7]
```diff
@@ -122,6 +122,81 @@ You can remove them by running the `delete-sensor.sh` script from the Sensor ins
...
+====
+When you do not specify the `-o` option for the output format:
+
+* The default output format for the `deployment check` and the `image check` commands is `table`.
+* The default output format for the `image scan` command is `json`. This is the old JSON format output for compatibility with older versions of the CLI. To get the output in the new JSON format, specify the option with format, as `-o json`.
+====
+
+Depending on the format you select, there are different options available to configure the output. The following tabele lists the options for each format.
+
+[%header,cols="1,1,2"]
```
Comment (kalexand-rh) [inline modules/using-cli.adoc:151]: This is a complex table with a fair amount of duplication. The complexity isn't the best from an accessibility standpoint. Consider reworking the way the information is presented in the table someday.

**Review by kalexand-rh — COMMENTED** [review]
I have a few picks and suggestions.

**Review by kalexand-rh — COMMENTED** [review]
I have a few picks and suggestions.

**Comment by gaurav-nelson** [comment]
Thank you @kalexand-rh I've accepted you suggestions and updated the table to combine common options. 👍🏼

**Pull request merged**